server/anomaly_scanner.go

```diff
@@ -349,7 +349,7 @@ func (s *AnomalyScanner) checkDatabaseAnomaly(ctx context.Context, instance *api
 			} else {
 				err := s.server.store.ArchiveAnomaly(ctx, &api.AnomalyArchive{
 					DatabaseID: &database.ID,
-					Type:       api.AnomalyDatabaseConnection,
+					Type:       api.AnomalyDatabaseSchemaDrift,
```

Comment:
Good catch and it looks like a bug. Let’s move this line change to a separate PR.

Reply:
Reverted and created a new PR: https://github.com/bytebase/bytebase/pull/2755.